libs/typescript-project-references/src/generators/migrate/generator.ts

```diff
@@ -160,6 +165,9 @@ function createTypeScriptConfig(
                 types: ['jest', 'node'],
             },
             include: ['src/**/*.ts', 'src/**/*.tsx'],
+            references: graph.dependencies[name].map((dep) => ({
```

Comment:
This is computed twice. Also I was expecting a filter as well but I guess the undefineds are filtered later with a json encode somewhere?

Reply:
Fixed